langchain4j/src/main/java/dev/langchain4j/service/DefaultAiServices.java

```diff
@@ -269,6 +279,35 @@ private static ChatMessage prepareUserMessage(Method method, Object[] args) {
         throw illegalConfiguration("For methods with multiple parameters, each parameter must be annotated with @V, @UserMessage, @UserName or @MemoryId");
     }
 
+    private static String getPromptText(Method method, String type, String resource, String[] value, String delimiter) {
+        String systemMessageTemplate;
```

Comment:
```suggestion
        String messageTemplate;
```
This method is appicable for user message as well